src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -657,6 +667,37 @@ private void DocumentFileChangeContext_FileChanged(object sender, string fullFil
             _additionalFiles.ProcessFileChange(fullFilePath);
         }
 
+        private void FileReferenceChangeContext_FileChanged(object sender, string fullFilePath)
```

Comment:
At one point I added a 5 second delay before re-reading the file, because for large assemblies it often remains locked through the first file change notification.  Is that still a concern?

Reply:
@Pilchie It possibly still is, I wasn't necessarily going to take that in this one for risk reduction unless you think we still need that here.

Reply:
That other change ended up being a servicing change for VS2015 IIRC, so I would strongly consider not going without it.

Reply:
@Pilchie I've made the update. If you can re-review that'd be appreciated.